### ISSUE TITLE
Final_review jx266

### ISSUE THREAD

**Issue by jx266** [Santostang/box-office-prediction#14]
The topic of this project is to predict the box office performance of the opening weekend prior to its release date. I like the way you express where and how you find different dataset. The example for google trend is useful and interpretable. There are 4 data source websites and you listed all the feature and their description in your report. Although it is very detailed and helpful for people who really want know every feature in this project, I think the explanation here is too much. Compared to the name and description of features, I prefer to know the details of why you choose column mean to fill the missing values at Dirctor_gross column. Because you said the data in this column was continuous, why not use the exited column to predict the missing values? And I think it will be better if you can test if your model with all the features is overfitting. Some of features have strong linear correlation with open_gross, maybe you can drop some features which have the similar change with open_gross to reduce the complexity of your model.

I think your team did excellent work on explaining your models. Quadratic loss functions with l1 and l2 regularizations and Huber loss function with l2 regularization are discussed (I am a little confused why not discuss Huber loss function with l1 regularization). Your explanation on how and why predicted opening gross is different from opening gross is convinced. The next test you did was to predict the gross one week before the release date and the better model was unchanged. In general, this project is very interesting and your results can well predict the target. It is a nice report!

**Comment by Santostang** [comment]
Thank you for your comment. I really appreciate your praise, it makes our efforts pay off. Here the reason we use mean value because it is commonly used and we did not have much time to predict the missing value. For similar features, we think l1 regularizers can do a good job on filter out them by fitting the model. Actually I am not sure that we need to drop features or keep them in this occasions.
